patches/react-native+0.72.6.patch

```diff
@@ -1,25 +1,30 @@
 diff --git a/node_modules/react-native/React/CoreModules/RCTTiming.mm b/node_modules/react-native/React/CoreModules/RCTTiming.mm
-index 13d0d57..00e5d4c 100644
+index e4f7e15..6f05fb3 100644
 --- a/node_modules/react-native/React/CoreModules/RCTTiming.mm
 +++ b/node_modules/react-native/React/CoreModules/RCTTiming.mm
-@@ -127,7 +127,15 @@ - (void)setup
+@@ -127,13 +127,16 @@ RCT_EXPORT_MODULE()
  {
    _paused = YES;
    _timers = [NSMutableDictionary new];
 -  _inBackground = NO;
-+    
+-  RCTExecuteOnMainQueue(^{
+-    if (!self->_inBackground && [RCTSharedApplication() applicationState] == UIApplicationStateBackground) {
+-      [self appDidMoveToBackground];
+-    }
++
 +  __block BOOL initialInBackground;
 +  dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
We need to update the patch here. They are alrey running the check on the main queue above, which is great. It just needs to include the check for the proximity state.

Reply:
I submitted out patch upstream: https://github.com/facebook/react-native/pull/41262 Perhaps you can just take that.